doc/examples/box2d/sprite_body/.gitignore

```diff
@@ -0,0 +1,70 @@
+# Miscellaneous
```

Comment:
do we still need this file? I thought @renancaraujo made a generic one for all examples